css/common.css

```
@@ -0,0 +1,70 @@
+body {
```

charset の指定を追加してもらえればこちらOKです〜👍
```suggestion
@charset 'utf-8' ;

body {
```